avocado/plugins/multiplex.py

```diff
@@ -48,7 +48,7 @@ def configure(self, parser):
 
         parser.add_argument('--filter-out', nargs='*', default=[],
                             help='Filter out path(s) from multiplexing')
-        parser.add_argument('-s', '--system-wide', action='store_true',
+        parser.add_argument('-sw', '--system-wide', action='store_true',
```

Comment:
Please don't use double character shortcuts. `-sw` should be the same as `-s` + `-w`. Making it a single shortcut breaks this assumption and is quite confusing.

When you have too many single-letter shortcuts, the solution is to use only long versions. There's nothing wrong with having `--system-wide` only, without a `-s` counterpart.